fsd_config/manifest.yml

```diff
@@ -17,6 +17,7 @@ applications:
       MULTIFUND_URL: https://frontend.dev.gids.dev/account
       BASIC_AUTH_ON: true
       CONTACT_US_URL: https://frontend.dev.gids.dev/contact_us
+      FEEDBACK_LINK: https://frontend.dev.gids.dev/feedback
```

Comment:
I'm confused as to why this is in this commit...

Reply:
Earlier we have manually added it test/dev . So now doing it the right way

Reply:
Ahhh, backwards fix. Regrettable but understandable. Happy with that